### ISSUE TITLE
Working Version

### ISSUE THREAD

**Issue by DifferentlyAbled** [d08115858/DropIt#1]
Hello, "Drop It" looks like a brilliant idea for a social media app. Do you happen to have a working Xcode project to take a look at? It seems like all of the Pods files are missing.

**Comment by MosesEsan** [comment]
Hi, I haven't actually worked on this project in about a year or so. I plan on taking some time in the next few weeks to update the code.

**Comment by Chazb123** [comment]
Yes, I love what you have done, it seems that every file under articles in the view controller folder are all missing. Is there any chance you could them to the project, thanks.
